Procfile

```diff
@@ -0,0 +1,4 @@
+# Run Rails & Webpack concurrently
+# Example file from webpack-rails gem
+rails: bundle exec rails server
```

Comment:
Could this have `-b 0.0.0.0` at the end since I think that will be the most common way to want to run this?